buildkite.yaml

```diff
@@ -6,8 +6,8 @@ steps:
       - "npm test"
     plugins:
       - docker#v3.7.0:
-          image: node:lts
+          image: cypress/base:16.13.0
```

Comment:
Hi @jesseduffield unfortunately this breaks the CI pipeline
https://buildkite.com/buildkite/javascript-test-collectors/builds/203#018b2121-524b-463a-9e5f-c5e4f63e6510.
Is there any docker image that we can use to avoid this? Or maybe we can create a separate step for Cypress, so it doesn't affect the existing step?

Reply:
I've switched back to node:lts and added a step for adding cypress dependencies